config/locales/en/controllers/messages.yml

```
@@ -0,0 +1,9 @@
+---
+en:
+  messages:
+    controller:
+      create:
+        sent_successfully: Message was successfully sent.
+      destroy:
+        error: An error occured while deleting the message.
```

```suggestion
        error: An error occurred while deleting the message.
```

There are a few files (translations), where my IDE points out some mistakes (such as here). Would you do me a favour and fix these, too, please? Just checking with the IDE should be fine, we don't need to rephrase everything 🙂

---

I used the typo checker and fixed some other mistakes too.